src/user/test/user.service.spec.ts

```diff
@@ -28,15 +28,16 @@ describe('UserService', () => {
   describe('CreateUser', () => {
     it('should create default User', async () => {
       //given
-      const request = createUser();
+      const { email } = createUser();
+      const password = faker.internet.password({ length: 64 });
 
       //when
-      const createdUser = await userService.createUser(request);
+      const createdUser = await userService.createUser({ email, password });
```

Comment:
hmm, why did we remove the `password` from `createUser` if we need it anyway?

Reply:
I used this function a lot while creating data for auth tests. But now since the `db` here (of base service) does not contain user password, and most of the endpoints did not required password after that change I removed it.